configs/pcx86/machine/compaq/lte286/cga/debugger/machine.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="/configs/pcx86/xsl/machine.xsl"?>
+<machine id="compaq-lte286-640k" type="pcx86" border="1" pos="center" background="default">
+	<name pos="center">COMPAQ LTE/286, 640Kb RAM</name>
+	<computer id="compaq-lte286-640k" name="COMPAQ LTE/286" buswidth="24"/>
+	<cpu id="cpu286" model="80286" cycles="12000000"/>
+	<ram id="ramLow" addr="0x00000" size="0xa0000"/>
+	<rom id="romBIOS" addr="0xf0000" size="0x10000" alias="0xff0000" file="/machines/pcx86/compaq/lte286/rom/1989-09-07/1989-09-07.json"/>
```

Comment:
Any idea if this ROM is *really* aliased at 0xff0000?

Reply:
> Any idea if this ROM is _really_ aliased at 0xff0000?

@jeffpar The Tech Ref said as much.   It will be interesting to see the debug dump once I get it.